cmd/restic/cmd_backup.go

```diff
@@ -532,8 +532,12 @@ func runBackup(opts BackupOptions, gopts GlobalOptions, term *termstatus.Termina
 		return err
 	}
 
-	if !gopts.JSON && parentSnapshotID != nil {
-		p.V("using parent snapshot %v\n", parentSnapshotID.Str())
+	if !gopts.JSON {
+		if parentSnapshotID != nil {
+			p.P("using parent snapshot %v\n", parentSnapshotID.Str())
+		} else {
+			p.P("no parent snapshot, need to read all data\n")
```

Comment:
I suggest "no parent snapshot found, will read all data\n".

Reply:
Or maybe "I suggest "no parent snapshot found, will read all files\n" is more relatable to users.